docs/source/gbd2019_models/hemoglobin_anemia_and_iron_deficiency/risk_effects/index.rst

```diff
@@ -35,3 +35,190 @@ Iron Deficiency Risk Effects
 .. contents::
    :local:
    :depth: 2
+
+GBD 2019 Modeling Strategy
+--------------------------
+
+Iron deficiency in the GBD risk factors is defined as **inadequate iron to meet the body's needs** and is quantified in terms of mean hemoglobin concentration at the population level from the cumulative effect of all causes that lead to iron deficiency, including dietary iron deficiency (GBD *cause* of inadequate intake of elemental iron) as well as other causes that manifest as iron deficiency (eg, maternal hemorrhage, uterine fibroids, menstrual disorders, hookworm, schistosomiasis, gastritis and duodenitis, inflammatory bowel disease, etc.). The iron deficiency risk exposure is defined as a **continuous distribution of hemoglobin concentrations in g/L** and is a risk factor for maternal disorders and has a risk-attributable cause of dietary iron deficiency.
+
+.. image:: iron_risk_hierarchy.svg
+
+.. list-table:: Affected Entities
+   :widths: 5 5 5 5 5
+   :header-rows: 1
+
+   * - Outcome
+     - Outcome type
+     - Outcome ID
+     - Affected measure
+     - Note
+   * - Maternal disorders
+     - Cause
+     - 366
+     - YLDs/YLLs
+     - Maternal disorders is not a most-detailed cause; the iron deficiency risk factor affects each of the most-detailed causes in the maternal disorders cause group equally
+   * - Dietary iron deficiency
+     - Cause
+     - 390
+     - Prevalence
+     - PAF-of-1 relationship
+
+Maternal Disorders
+++++++++++++++++++
+
+The GBD 2019 Risk-Outcome pair relationship between iron deficiency and maternal disorders relies on the association between hemoglobin concentration and maternal mortality from the literature, specifically the CHERG iron report.
+
+.. todo::
+
+  Cite the CHERG iron report, GBD 2019 methods appendix
+
+.. note::
+
+  According to a correspondence with Chris Murray, there is contraversy surrounding the inclusion of this risk-outcome pair in GBD because there is evidence supporting an association between iron supplementation and hemoglobin increases as well as evidence supporting an association between hemoglobin increases and maternal mortality, but no/little evidence to support an association between iron supplementation and maternal mortality directly.
+
+Notably, the *same* relative risks for this risk-outcome pair apply to both YLDs and YLLs and are constant for each age-group and each most-detailed cause within the maternal disorders cause heirarchy; assuming that changes in hemoglobin concentration affect each of the maternal disorders equally and also affect morbidity and mortality equally.
```

Comment:
This sentence is a bit confusing to me, as the clause after the semicolon is not a complete sentence.  Is this second part just restating what the part before the semicolon means?